### DISCUSSION TITLE
How can I set typeCheckingMode = "off" in lunarvim?

### DISCUSSION THREAD

**Discussion by PCesteban** [LunarVim/LunarVim#2857] [Q&A] [answered]
How can I do this: 
![image](https://user-images.githubusercontent.com/48070832/181115003-328f7b4e-febe-478d-a7a3-7b069e34de63.png)

in lunarvim config file? I know that lspconfig is a built-in plugin.

**Comment by rebuilt** [comment, marked as answer]
Check out the section about overriding the default lsp setup.   https://www.lunarvim.org/languages/#server-setup
